packages/info/logger/mixin.server.js

```diff
@@ -0,0 +1,35 @@
+const { sync } = require('mixinable');
+const { Mixin } = require('hops-mixin');
+const { internal: bootstrap } = require('hops-bootstrap');
+const { createLogger } = require('.');
+
+const { callable } = sync;
+const { validate, invariant } = bootstrap;
+
+class ServerLogMixin extends Mixin {
+  constructor(...args) {
+    super(...args);
+
+    const [{ name, rootDir }] = args;
+
+    this.logger = createLogger({
+      name: `${name}:runtime`,
+      _workspace: rootDir,
```

Comment:
The `_workspace`-value from the config is used to cut pathnames in error stacks. But this value isn't available here. So the workaround is to use the `rootDir`, which might lead to non-truncated pathnames in a setup comprised of workspaces, because those values differ in this setup.

I'm open for suggestions on how to do this better! 😃